### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by jayvdb** [venmo/nose-detecthttp#2]
The plugin fails to load on Python 3 

```bash
$ nosetests --version
.../.tox/nose34/lib/python3.4/site-packages/nose/plugins/manager.py:395: RuntimeWarning: Unable to load plugin detecthttp = detecthttp:DetectHTTP: No module named 'plugin'
nosetests version 1.3.7
```

**Comment by simon-weber** [comment]
Thanks for the heads up. Unfortunately, I'm unlikely to find the time to make this dual compatible anytime soon. So definitely send a pull request if you end up handling it :grin:

**Comment by jayvdb** [comment]
It worked for me on Python 3 with PR #3 (I also needed to patch vcr https://github.com/kevin1024/vcrpy/pull/223 )

**Comment by jayvdb** [comment]
Any chance for a release?

**Comment by simon-weber** [comment]
Sure. I'll do that today.

**Comment by simon-weber** [comment]
All set: https://pypi.python.org/pypi/nose-detecthttp/0.1.3.

**Comment by jayvdb** [comment]
And verified fixed on Python 3.4.  Much appreciated.

**Comment by simon-weber** [comment]
Sure thing; thanks for letting me know :+1:
